numpyro/distributions/discrete.py

```diff
@@ -437,6 +437,9 @@ class PRNGIdentity(Distribution):
     is_discrete = True
 
     def __init__(self):
+        warnings.warn("PRNGIdentity distribution is deprecated. To get a random "
+                      "PRNG key, you can use `numpyro.prng_key()` instead.",
+                      DeprecationWarning)
```

Comment:
Sorry, I didn't notice this. Could you change this to `FutureWarning` instead? See https://docs.python.org/3/library/warnings.html#warning-categories.

Reply:
Oh, thanks! You keep reminding me about this but I forgot. >__<